dev-manuals/test-plans/minigames/regex-game-frontend/plans/u-regex-game-3-highscore-not-update.md

```diff
@@ -0,0 +1,43 @@
+# Regex Game: Only update highscore when `score > highscore` (`u.regex-game-3`)
+
+Version: V1.1, 09.08.2022, Improve format, merge with `score <= highscore does not update highscore` test \
+Author: Leon Layer, Leon Hofmeister
+
+## Description
+
+Tests that the highscore doesn't change when `score <= highscore`.
+
+## Precondition
+
+The highscore is `>= 2`. The game has at least three regexes. The current score is `0`.
```

Comment:
maybe don't use concrete numbers here

Reply:
Why? I mentioned in the alternate procedures how to handle it if we change our scale.